src/plots/cartesian/select.js

```diff
@@ -192,6 +192,8 @@ function prepSelect(e, startX, startY, dragOptions, mode) {
         }
     }
 
+    var direction = fullLayout.selectdirection;
```

Comment:
`direction` doesn't get used in the outer scope. Could be move that `var direction` into the `moveFn` scope below?

Reply:
good call -> pushed to inner scope https://github.com/plotly/plotly.js/pull/2506/commits/9dd166c69bd654bd0a70dac6d2f0cb4f3d6c7235